cmodelres/CMakeLists.txt

```diff
@@ -42,6 +42,31 @@ else()
     message(WARNING "Failed to find sea. See -DSEA_PATH.")
 endif()
 
+# Locates KLEE, as required for testing.
```

Comment:
@huxinwen1994 I assume that this is just adapted from the way we locate Seahorn. Right?

Reply:
yes